src/main/resources/static/swagger.yaml

```diff
@@ -440,4 +504,30 @@ components:
           example: Curve25519/32byte random key
         keyValue:
           type: string
-          format: string
\ No newline at end of file
+          format: string
+    SessionRequest:
+      type: object
+      required:
```

Comment:
Should also take in "display".  We should use this "display" element to show the Doctor's or any Users name.

Reply:
I think it's okay to show username as display name right now. It's mock UI we are developing.